src/jdk.incubator.code/share/classes/jdk/incubator/code/dialect/core/CoreOp.java

```diff
@@ -123,6 +120,13 @@ public FuncOp transform(OpTransformer ot) {
             return new FuncOp(this, CopyContext.create(), ot);
         }
 
+        FuncOp(FuncOp that, String funcName, CopyContext cc, OpTransformer ot) {
```

Comment:
Group that with the other copying constructor, then its easier to see their relationship and whether we need to keep both.